bin/codecov.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -e
+set -u
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
+source $SCRIPTPATH/use_bazel_go.sh
+
+ROOTDIR=$SCRIPTPATH/..
+cd $ROOTDIR
+
+
+echo "Code coverage test"
+echo "" > coverage.txt
+for d in $(go list ./... | grep -v vendor); do
+    go test -coverprofile=profile.out $d
```

Comment:
we could do this as part of cirlceci, because we are already testing pilot, broker, security. And you don't need tokens from circleci